mlflow/tracking/fluent.py

```diff
@@ -1020,3 +1036,26 @@ def _get_experiment_id():
         or _get_experiment_id_from_env()
         or (is_in_databricks_notebook() and get_notebook_id())
     ) or deprecated_default_exp_id
+
+
+def universal_autolog(
```

Comment:
In accordance with the comment in `mlflow/__init__.py`, can we rename this to `autolog()` for consistency with other modules & for code discoverability?